T/TestU01/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "TestU01"
+version = v"1.2.3"
+
+# Use unofficial git mirror instead of official URL, as it is unversioned and may be overwritten in the future.
+# Manually verified to be bit-for-bit identical (except for a missing empty `bin/` directory) to official v1.2.3 release.
+sources = [
+    GitSource(
+        "https://github.com/blep/TestU01.git",
```

Comment:
The official git repo is https://github.com/umontreal-simul/TestU01-2009 but there are no releases or tags.

Reply:
Doh. I should just read the comment. Never mind.